bin/smarthome.py

```diff
@@ -50,7 +50,7 @@
 #####################################################################
 # Base
 #####################################################################
-BASE = '/'.join(os.path.realpath(__file__).split('/')[:-2])
+BASE = os.path.sep.join(os.path.realpath(__file__).split(os.path.sep)[:-2])
```

Comment:
I guess, this could also be done by using `os.path.dirname(os.path.dirname(os.path.realpath(__file__)))`. No need to do join/split using the separator at all.